tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1274,15 +1249,20 @@ namespace TensileLite
         if(gsu > 0)
             rv.numWorkGroups.y *= gsu;
 
-        size_t skGrid    = 0;
-        auto   tiles     = problem.getNumTiles(sizeMapping);
+        size_t skGrid = 0;
+        auto   tiles  = problem.getNumTiles(sizeMapping);
         if(sizeMapping.streamK != 0 || sizeMapping.persistentKernel != 0)
         {
             AMDGPU const* pAMDGPU = dynamic_cast<AMDGPU const*>(&hardware);
             assert(pAMDGPU != nullptr && pAMDGPU->computeUnitCount != 0);
             if(sizeMapping.streamK != 0)
             {
-                skGrid             = getSKGrid(problem, hardware, tiles);
+                const bool streamKDP = Debug::Instance().useStreamKDataParrallel();
+                if(streamKDP)
+                    skGrid = tiles;
+                    //TODO Use heuristic to decide fallback to reduced grid instead of DP
+                else
```

Comment:
Might be safer to include this check inside getSKGrid, since that is called from a couple places